Cargo.toml

```
@@ -143,4 +143,7 @@ name = "world_basic"
 
 [[example]]
 name = "world_chunking"
-required-features = ["debug"]
\ No newline at end of file
+required-features = ["debug"]
+
+[patch.crates-io]
+bevy_ecs_tilemap = { git = "https://github.com/shanecelis/bevy_ecs_tilemap.git", branch = "add-pivot" }
```

Now that your PR is merged, you can use bevy_ecs_tilemap `main` branch:

```suggestion
bevy_ecs_tilemap = { git = "https://github.com/stararawn/bevy_ecs_tilemap.git", rev = "037b0282cd1ac85802b15db1887642e26d9c6f7b" }
```

---

I think the main branch is going to be supporting bevy 0.16, which is why I was hesitant to use it.

---

Yes, but this particular commit is just before 0.16 changes